ramalama/common.py

```diff
@@ -102,7 +102,7 @@ def find_working_directory():
 def run_curl_cmd(args, filename):
```

Comment:
We can actually delete this whole function, it's not called anywhere anymore, we move to a python based http client, so we could have prettier progress bars, etc. and one less dependancy.

Reply:
Nice, next PR.